recipes/mender-connect/debian-master/control

```diff
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 9)
 
 Package: mender-connect
 Architecture: any
-Depends: mender-client (>3.2.0), ${shlibs:Depends}, ${misc:Depends}
+Depends: mender-client (>4.0.0), ${shlibs:Depends}, ${misc:Depends}
```

Comment:
mender-auth, not mender-client, right?

Reply:
From [this page](https://www.debian.org/doc/debian-policy/ch-relationships.html):

> the package names listed may also include lists of alternative package names, separated by vertical bar (pipe) symbols

Sounds exactly like what we need, doesn't it?